index.php

```diff
@@ -24,6 +24,6 @@
 
 $app = new Application($ROOT_DIR, $configuration);
 
-require_once Application::$ROOT_DIR . '/router/route.php';
+require_once $app::$ROOT_DIR . '/router/route.php';
```

Comment:
I think you need to check if the file will be check on view folder in case they used an other fool path

Reply:
These is #95  that will managed thoses situation.